scripts/oc-cluster.sh

```diff
@@ -12,15 +15,15 @@ sudo service docker start
 sudo service docker status
 
 ## download oc binaries
-sudo wget https://github.com/openshift/origin/releases/download/v3.10.0/openshift-origin-client-tools-v3.10.0-dd10d17-linux-64bit.tar.gz -O /tmp/openshift-origin-client-tools.tar.gz 2> /dev/null > /dev/null
+sudo wget $OPENSHIFT_CLIENT_BINARY_URL -O /tmp/openshift-origin-client-tools.tar.gz 2> /dev/null > /dev/null
 
 sudo tar -xvzf /tmp/openshift-origin-client-tools.tar.gz --strip-components=1 -C /usr/local/bin
 
 ## Get oc version
 oc version
 
 ## below cmd is important to get oc working in ubuntu
-sudo docker run -v /:/rootfs -ti --rm --entrypoint=/bin/bash --privileged openshift/origin:v3.10.0 -c "mv /rootfs/bin/findmnt /rootfs/bin/findmnt.backup"
+sudo docker run -v /:/rootfs -ti --rm --entrypoint=/bin/bash --privileged openshift/origin:`echo $OPENSHIFT_CLIENT_BINARY_URL | awk -F '//' '{print $2}' | cut -d '/' -f 6` -c "mv /rootfs/bin/findmnt /rootfs/bin/findmnt.backup"
```

Comment:
might wanna move this into a variable `OPENSHIFT_CLIENT_LATEST_VERSION`

Reply:
Good choice!!!
Done